pyproject.toml

```
@@ -8,7 +8,7 @@ description = "PennyLane-Lightning plugin"
 readme = "README.rst"
 requires-python = ">=3.10"
 classifiers = [ "Development Status :: 4 - Beta", "Environment :: Console", "Intended Audience :: Science/Research", "License :: OSI Approved :: Apache Software License", "Natural Language :: English", "Operating System :: MacOS :: MacOS X", "Operating System :: Microsoft :: Windows", "Operating System :: POSIX", "Operating System :: POSIX :: Linux", "Programming Language :: Python", "Programming Language :: Python :: 3", "Programming Language :: Python :: 3 :: Only", "Programming Language :: Python :: 3.10", "Programming Language :: Python :: 3.11", "Programming Language :: Python :: 3.12", "Topic :: Scientific/Engineering :: Physics",]
-dependencies = ["pennylane>=0.37"]
+dependencies = ["pennylane>=0.37", "scipy-openblas32"]
```

For the new symbols and methods naming convention, we should pin to this: 
```suggestion
dependencies = ["pennylane>=0.37", "scipy-openblas32>=0.3.26"]
```